### ISSUE TITLE
[ATTENTION CONTRIBUTORS] project has been forked to tgenv/tgenv for ongoing maintenance

### ISSUE THREAD

**Issue by weyderfs** [cunymatthieu/tgenv#45]
Guys, apparently, this project is dead.

So I propose to fork this project and continue with the improvements that we have through this new fork, but I will need the help of all of you, actually there's the 14 _pull-requests_  and 16 _issues_ opens, will be interesting move that to new project too.

What do you guys think about?

I ask you to react with 👍🏻  here.

**Comment by iokiwi** [comment]
Yes should we make an organization with a more canonically definite name such as `tgenv/tgenv`.

**Comment by iokiwi** [comment]
There's also the question of what to do with packages published to repositories such has homebrew

  * https://formulae.brew.sh/formula/tgenv
  
 If the original author is no longer able or interested in maintaining this repository maybe they would like to hand over the reigns? Or, alternatively, would @gruntwork-io consider taking this under their umbrella?

**Comment by weyderfs** [comment]
> Yes should we make an organization with a more canonically definite name such as `tgenv/tgenv`.

Good point @iokiwi , It's a good way to follow. Are you interested in maintainer de project?

---

> There's also the question of what to do with packages published to repositories such has homebrew
> 
>     * https://formulae.brew.sh/formula/tgenv
> 
> 
> If the original author is no longer able or interested in maintaining this repository maybe they would like to hand over the reigns? Or, alternatively, would @gruntwork-io consider taking this under their umbrella?

I don't know about how works the homebrew workflows, could it be that we were unable to put the new formula, pointing to the new project?

Maybe other option is launch with a new brew formule like `tgenv-manager`

**Comment by iokiwi** [comment]
I could be a maintainer if need be but I think the important thing is having shared ownership / maintainer ship so that neither burden nor the keys to the kingdom fall to one individual and we end up back in this situation.

**Comment by iokiwi** [comment]
I guess we can deal with the homebrew issue later as long as people can install from source.

@cunymatthieu how would you feel about adding some more maintainers/letting us revive this project?

**Comment by weyderfs** [comment]
@iokiwi I've created the Organization.

About the _homebrew_,  I agree with you, this can be handled in the future.

I saw that you mention @cunymatthieu , I just started this _issue/thread_, because he hadn't been replying... 😅😅😅😅

**Comment by weyderfs** [comment]
So, I've created the organization and forked the project [tgenv/tgeng](https://github.com/tgenv/tgenv).

Everyone is welcome to help and maintainer the project.

**Comment by iokiwi** [comment]
Hey should we update the title of this issue to reflect that this is no longer just a proposal but has infact been revived at tgenv/tgenv?

Something along the lines of:

>[Moved] project has been forked to tgenv/tgenv for ongoing maintenance

**Comment by weyderfs** [comment]
@iokiwi good suggestion, done.

**Comment by iokiwi** [comment]
Recon we should change the title again to `[ATTENTION CONTRIBUTORS]` are you please able to do that @weyderfs

**Comment by weyderfs** [comment]
> Recon we should change the title again to `[ATTENTION CONTRIBUTORS]` are you please able to do that @weyderfs

Done, bro! 📍
